app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -53,7 +53,7 @@ if [ x$ID = xrhel ] || [ x$ID = xcentos ]; then
     }
 
     CONSUMER_RPM=$(mktemp --suffix .rpm)
-    curl --output $CONSUMER_RPM <%= subscription_manager_configuration_url %>
+    curl --output $CONSUMER_RPM <%= subscription_manager_configuration_url(nil, true, @url_host) %>
```

Comment:
Is this allowed syntax in safemode?
```suggestion
    curl --output $CONSUMER_RPM <%= subscription_manager_configuration_url(hostname: @url_host) %>
```

Reply:
But:
```ruby
def test(one = nil, two = true, three = 'something')
  puts one
  puts two
  puts three
end

# Run test()
nil
true
something

# Run test(three: 'text')
{:three=>"text"}
true
something
```

Reply:
Ah yes, you need to make it callable like that on the method
```ruby
def test(one = nil, two = true, three: 'something')
  puts one
  puts two
  puts three
end

# Run test()
nil
true
something

# Run test(three: 'text')
nil
true
text
```